source/pymongo.txt

```diff
@@ -76,10 +75,24 @@ Connect to MongoDB Atlas
 
    import pymongo
 
-   client = pymongo.MongoClient(
-      "mongodb+srv://<username>:<password>@<cluster-url>/test?retryWrites=true&w=majority")
-   db = client.test
+   // replace this with your MongoDB connection string
+   conn_str = "<your MongoDB Atlas connection string>"
+
```

Comment:
Suggestion: Since you mention the timeout after the snippet, it would be beneficial to mention that it's being created here with a comment

Also applies to Motor page.

Reply:
Sounds good if you think it helps the reader.

I intentionally omitted that since I wanted these to be as brief as possible. To me, the goal of the connection snippet on landing pages is to be as brief as possible and not serve as a quick start. For example, we don't explain the connection string format, the client, the command to test the connection, nor output on the other drivers. I wanted to change Python drivers so that we gave slightly more guidance but kept it as short as possible, but this seems like a reasonable addition.